api/clients/v2/node_client.go

```diff
@@ -3,6 +3,7 @@ package clients
 import (
 	"context"
 	"fmt"
+	"github.com/Layr-Labs/eigenda/node/auth"
```

Comment:
It doesn't seem to me this is a good dependency direction for api/ to depend on node/

Reply:
this dependency is no longer present